packages/shylu/shylu_dd/frosch/src/Tools/FROSch_SubdomainSolver_def.hpp

```diff
@@ -69,8 +74,11 @@ namespace FROSch {
             EpetraLinearProblem_.reset(new Epetra_LinearProblem(epetraMat.get(),xTmp.get(),bTmp.get()));
             
             Amesos amesosFactory;
+
             AmesosSolver_.reset(amesosFactory.Create(ParameterList_->get("Solver","Mumps"),*EpetraLinearProblem_));
-            AmesosSolver_->SetParameters(ParameterList_->sublist("Amesos")); 
+
+            AmesosSolver_->SetParameters(ParameterList_->sublist("Amesos"));
```

Comment:
Did you already push the fix for the parameters for Amesos2?

Reply:
Added in new commit. Now the name of parameterlist passed in is "Amesos2" and not something like "...sublist->sublist->Amesos2"